LoopKit/GlucoseKit/GlucoseStore.swift

```diff
@@ -96,7 +96,11 @@ public final class GlucoseStore: HealthKitSampleStore {
     }
     private let lockedLatestGlucose = Locked<GlucoseSampleValue?>(nil)
 
-    private let cacheStore: PersistenceController
+    public let cacheStore: PersistenceController
+    
+    static let queryAnchorMetadataKey = "com.loopkit.GlucoseStore.queryAnchor"
+
+    private let startAfterDatePredicate = NSPredicate(format: "startDate >= $start")
```

Comment:
This is a "new" way to create this predicate. Looking at the single usage of `startAfterDatePredicate` I don't see why it wasn't left as-is.

Reply:
Guessing it might've been a small optimization, but I agree, not worth it here.